.github/workflows/ci.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: [ '7.4', '8.0','8.1','8.2' ]
+        php-versions: [ '8.0','8.1','8.2' ]
```

Comment:
was 't nodig om dit er uit te halen? Ik had 't zelf niet gedaan maar kan 🤷

Reply:
ik zou zelf de Radiation in een minor zetten, en dan losse PR voor php7.4 in een major

Reply:
Yeah, ik wilde graag ctor promotion.. Ik heb het voor nu wel even ongedaan gemaakt